rust/main/utils/run-locally/src/sealevel/termination_invariants.rs

```diff
@@ -67,6 +67,7 @@ pub fn termination_invariants_met(
         RELAYER_METRICS_PORT,
         total_messages_expected,
         &hashmap! {"chain" => "sealeveltest1", "connection" => "rpc", "status" => "success"},
+        &hashmap! {},
```

Comment:
how come this is empty? IIUC we _are_ counting SVM provider instances, assuming an empty hashmap means we're not actually reading the provider count metric

Reply:
For some reason, `sealeveltest1` was only instantiating a small amount of providers. that was less than the expected message count.

Reply:
ah makes sense, we need to use the sealevel message count instead